packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -407,7 +407,8 @@ export interface IContextualMenuItem {
     itemClassName?: string,
     dividerClassName?: string,
     iconClassName?: string,
-    subMenuClassName?: string) => IMenuItemClassNames;
+    subMenuClassName?: string,
+    primaryDisabled?: boolean) => IMenuItemClassNames;
```

Comment:
It's a little awkward to have this param at the end (and for it to be optional), but it is the only way to preserve backwards compat. The getStyles API solves this problem, but we haven't moved ContextualMenu to that API yet.